src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -1696,12 +1863,7 @@ def _handleScaleToVisibleAreaClicked(self):
         item = self._getItem()
         if item is None:
             return  # no-op
-
-        bounds = item.getVisibleBounds()
-        if bounds is None:
-            return  # no-op
-
```

Comment:
I guess that was here to handle the case the whole item is not visible (see other comment).